### ISSUE TITLE
training loop

### ISSUE THREAD

**Issue by buehlpa** [gordon-chenmo/UTRAD#3]
Hi , thanks for that great work & code implementation,

I just have a Question to the training process, For the MVTec dataset: Did you use just 150 epochs to train each model? as statet in the paper, or did you have another metric for the training loop to see when there is a "good" fit? How did you come up with the 150?

Thanks ,
Cheers Pascal
